circuitx/gesture-navigation/src/androidMain/kotlin/com/slack/circuitx/gesturenavigation/GestureNavigationDecoration.kt

```diff
@@ -87,7 +94,6 @@ private class AndroidPredictiveNavigationDecoration(
       }
 
       transition.AnimatedContent(
-        modifier = modifier,
```

Comment:
This was a bug 😬

Reply:
Maybe we should start running the compose-lints on this repo too heh